wiki_guidelines.md

```diff
@@ -0,0 +1,66 @@
+# Wiki contribution guidelines 
+
+## Purpose 
+The wiki is our main source of documentation on the project. For newcomers to the project as well as existing contributors, the wiki should contain comprehensive information about each aspect of the project. As the project grows, the wiki grows with it and is pivotal in the maintenance of the project. 
+
+## Wiki Structure 
+We have a tree structure for our wiki, with relevant pages bundled together under the same cateogry. 
+* In the Home page you will find a list of contributors to the project. 
+* In the home section, you will find our code of conduct while working within the team as well as the overall timeline of the project. 
+* In the team section there will be a list of meeting minutes as well as the contributions each team member has made. 
+* A best practices section describes our strategy for the git workflow, issue writing, pull request guidelines, commenting guidelines and other best practices developers contributing to the project should follow. 
+* The tech section describes the technology stack used in the project and the reasoning behind our approach. As well other implementation details pertaining to the project. 
+* A single future works page will contain work that has not been started but could be something that future contributors could work on.
+
+
+### Meeting minutes format
+Meeting minutes format should be standardised and should include:
+* Date
+* Agenda
+* Members present 
+* Summary of meeting
+
+### Documenting contributions 
+When documenting contributions to the project, each contributor should be fairly credited with the work they have accomplished. So the contributions page should include a short description of the different work each team member contributed (both technical or non-technical). The contributions page should be split up into the different parts of the project for example. 
+
+```
+DevOps 
+- Casey: I worked on x feature, and y feature and helped on z feature.
+- Reshad: I worked on m feature and helped with x feature 
+... 
+
+Frontend 
+- Jason: I helped build s feature and also worked on p feature
+- Lucy: I worked on p feature and w feature 
+...
+
+Backend
+...
+
+Documentation
+...
+
+And so on..
+```
+
```

Comment:
I kinda feel like contributors and contributions can be on the same page?

Reply:
@baja156 Contributions can get bloated so it might deserve to be on a separate page :)